octopoes/octopoes/xtdb/query.py

```diff
@@ -43,7 +43,6 @@ class Aliased:
 
 
 Ref = type[OOI] | Aliased
-A = Aliased
```

Comment:
This was not a typing issue, but I think we shouldn't create one-level classes but always use proper names...